assets/src/application/ui/display/EmptyState.tsx

```diff
@@ -12,11 +13,11 @@ interface EmptyStateProps {
 
 const EmptyState: React.FC<EmptyStateProps> = ({ children, description, title, ...props }) => {
 	const className = classNames(props.className, 'ee-empty-state');
-	const iconProps = {
+	const iconProps = useMemoStringify({
 		color: 'gray.100',
 		name: 'question-outline',
 		size: '96px',
-	};
+	});
```

Comment:
this was already extracted wouldn't be simpler if we'd pull this out of component ?